src/components/shared/formItems/ProjectHandle.tsx

```diff
@@ -54,21 +57,27 @@ export default function ProjectHandle({
     }
   }, [inputContents])
 
-  // InputContents pattern allows checking if handle exists while typing
+  // inputContents pattern allows checking if handle exists while typing
   const idForHandle = useContractReader<BigNumber>({
     contract: ContractName.Projects,
     functionName: 'projectFor',
     args: handle && requireState ? [handle] : null,
     callback: useCallback(
-      returnValue === 'id'
-        ? id => onValueChange(id?.toHexString() ?? '0x00')
-        : () => null,
-      [returnValue],
+      id => {
+        setHandleLoading(false)
```

Comment:
Remove the loading spinner once we get an `id` for the handle.

`suffix` (below) will display the correct help text in the input field.